site/docs-md/cordova/using-cordova-plugins.md

```diff
@@ -0,0 +1,51 @@
+# Using Cordova Plugins and Ionic Native
+
+When developing an app that uses Capacitor, it's possible to use both Cordova and Ionic Native plugins.
+
+## Installing Cordova Plugins
+
+Simply install your plugin of choice, sync your project, finish any required native project configuration, and you're ready to go:
+
+```bash
+npm install cordova-plugin-name
+npx cap sync
+```
+
+## Updating Cordova Plugins
```

Comment:
No, update command doesn't detect a new version of a plugin and install it, update just update the native project with the installed version of the plugins